index.html

```diff
@@ -4,17 +4,196 @@
+                    <div class="option-1">
+                        <div class="option-number">
+                            <img src="./images/juice-bar-images/delivery-option-1.png" alt="options">
+                        </div> 
+                        <h3>Local Delivery</h3>
+                        <p>I'm a paragraph. Click here to<br> add your own text and edit me.</p> 
+                    </div>
+
+                    <div class="option-2">
+                        <div class="option-number"><img src="./images/juice-bar-images/delivery-option-2.png" alt="options"></div> 
+                        <h3>In Store Pick Up</h3>
+                        <p>I'm a paragraph. Click here to<br> add your own text and edit me.</p> 
+                    </div>
+
+                    <div class="option-3">
+                        <div class="option-number"><img src="./images/juice-bar-images/delivery-option-3.png" alt="options"></div>
+                        <h3>Curbside Pick Up</h3>
+                        <p>I'm a paragraph. Click here to<br> add your own text and edit me.</p>
+                    </div>
+                </div>                  
+            </div>
+        </main>
+    <!-- footer     -->
+        <footer class="footer">
+            <div class="follow-us">                  
+                <h1>Follow Us</h1>
+            </div>
+            <div class="image-container">
+               <img class="img-1" src="./images/juice-bar-images/juice_bar_image_6.jpg" alt="Cutting apples photo">
+               <img class="img-2" src="./images/juice-bar-images/juice_bar_image_7.png" alt="Fruit cut in half photo">
+               <img class="img-3" src="./images/juice-bar-images/juice_bar_image_8.png" alt="A woman carrying fruit photo">
+               <img class="img-4" src="./images/juice-bar-images/juice_bar_image_9.png" alt="Lime juice in bottles photo">
+            </div>   
+            <div class="footer-middle">
+                <div class="footer-nav">
+                    <div class="footer-get-started">
+                       <h3>Get Started </h3>
+                       <ul>
+                           <li>Home</li>
+                           <li>Story</li>
+                           <li>Menu</li>
+                           <li>Order Online</li>
+                           <li>Location & Hours</li>
+                           <li>Contacts</li>       
+                       </ul>
+                    </div>
+                    <div class="footer-connect">
+                        <h3>Connect</h3>
+                        <ul>
+                            <li>Instagram</li>
+                            <li>Facebook</li>
+                            <li>Youtube</li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="footer-email">
+                    <h3>Join Our Mailing List</h3>
+                    <div class="email-submit" >
```

Comment:
remember to wrap these elements in the `form` element instead of a `div`

Reply:
Thanks Ellie, I've changed it.